docs/gettinginvolved/acceptance-testing.rst

```diff
@@ -133,3 +133,27 @@ You will need a ssh agent running with access to the corresponding private key.
 .. prompt:: bash $
 
   admin/run-acceptance-tests --distribution fedora-20 --provider aws --config-file config.yml
+
+
+DigitalOcean
+------------
+
+To run the acceptance on DigitalOcean, you need:
+
+- a DigitalOcean account,
+- a "Legacy API v1" Client ID and API key
+  (https://cloud.digitalocean.com/generate_api_key),
+- an "API v2" token, which will be used to update the kernel of new droplets,
+  (https://cloud.digitalocean.com/settings/applications), and
+- an ssh-key registered with the AWS account.
+
+.. code-block:: yaml
+
+   digitalocean:
+     client_id: "<DigitalOcean API v1 client id>"
+     api_key: "<DigitalOcean API v1 api key>"
+     token: "<DigitalOcean API v2 api token>"
+     location: "<DigitalOcean location slug e.g. lon1>"
```

Comment:
It's not easy to lookup the slugs except if you use the API or if you manually create a droplet and look at its location in the web interface; that will show up as it's slug rather than full location name. I added some more examples in the document, but I'm not going to document it any further because hopefully this will all be automated soon and the configuration will be stored in LastPass somewhere.